app/src/main/java/com/github/se/cyrcle/model/parking/Tile.kt

```diff
@@ -67,5 +68,28 @@ data class Tile(val bottomLeft: Point, val topRight: Point) {
               Point.fromLngLat(x * TILE_SIZE, y * TILE_SIZE),
               Point.fromLngLat((x + 1) * TILE_SIZE, (y + 1) * TILE_SIZE)))
     }
+
+    /**
+     * Return all tiles that are in the circle defined by the center and the radius.
+     *
+     * @param center the center of the circle
+     * @param radius the radius of the circle in meter
+     * @return a set of tiles that are in the circle
+     */
+    fun getAllTilesInCircle(center: Point, radius: Double): Set<Tile> {
+      val leftestPoint = TurfMeasurement.destination(center, radius, -90.0, "meters")
```

Comment:
i think you meant leftmost (resp rightmost) point. Also it's not very consistent how you say "leftmost" but not "topmost", just "top"

Reply:
Ok, I'll go with the cardinal then. northPoint, eastPoint ...